brave/src/main/java/com/linecorp/armeria/internal/common/brave/SpanConverter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.common.brave;
+
+import com.linecorp.armeria.common.logging.TracingContext;
+
+import brave.Span;
+import brave.propagation.TraceContext;
+
+public final class SpanConverter {
+
+    public static TracingContext toLogTracingContext(Span span) {
+        final TraceContext traceContext = span.context();
+
+        return TracingContext.of(traceContext.traceIdHigh(), traceContext.traceId(),
```

Comment:
should add a benchmark if this will be used in a hot path, to make sure any overhead is fine. IIRC brave caches the string versions of IDs, but if nothing uses these data it would be useless overhead

Reply:
That's a good idea. 👍 
@codefromthecrypt How do you feel about this comment because you are the expert on distributed tracing? https://github.com/line/armeria/pull/5414#discussion_r1488938475
Can all tracing frameworks and libraries share the `TracingContext`?

Reply:
there are differences between libraries. For example, the localRoot is not a common concept, it is only available in brave (unless someone copied it later). also some don't propagate parent ID. I think trace and span ID are pretty common though.

Reply:
brave has built-in context stuff, and maybe otel does too? not sure. It might be better to configure this stuff per library vs make an abstraction https://github.com/openzipkin/brave/tree/master/context/slf4j

Reply:
Thanks @codefromthecrypt for the suggestion. 😉 
Then, I think we'd better put this in the brave module and abstract it later. @KonaEspresso94 Could you do that, please?

Reply:
yep, actually the context integration was some of the trickiest code, and thankfully had a good design bud @anuraaga at the time

Reply:
@KonaEspresso94 Please forget about my previous comments. 😓 
Let's add `TracingContext` to the core module as you did.
Before proceeding, as suggested by @codefromthecrypt, it's worth noting that not all properties in the TracingContext are common to other tracing frameworks and libraries. Could you conduct research on other platforms such as [OpenTelemetry](https://opentelemetry.io/) and [Micrometer tracing](https://docs.micrometer.io/tracing/reference/) to identify common properties?